hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -555,11 +592,30 @@ public void findOrRegisterTargetIfItDoesNotExistThrowsExceptionAfterMaxRetries()
         }
     }
 
+    @Test
+    @Description("Register a controller which does not exist, when a ConcurrencyFailureException is raised, the "
+            + "exception is rethrown after max retries")
+    public void findOrRegisterTargetIfItDoesNotExistWithNameThrowsExceptionAfterMaxRetries() {
```

Comment:
remove this test, because it does not make sense due to `findOrRegisterTargetIfItDoesNotExist` overloading

Reply:
Removed.